### ISSUE TITLE
[Question] How to broadcast message to all devices?

### ISSUE THREAD

**Issue by lincoln310** [thingsboard/thingsboard#7460]
**Component**

 * Generic

**Description**
For example, system need to send alarm to all device when some error happend.

**Environment**

 * OS:  all
 * ThingsBoard: 3.4.2
 * Browser: all

**Comment by ChrisEvisi** [comment]
Yes, PE has a 'Duplicate to Groups' rulenode which is like broadcasting message to all entities in an Entity Group.
